### PR TITLE
DE463738 : Fixed the environmentIncluded flag issue for full bundle

### PR DESCRIPTION
**Pull request by RaghuReddy**
Fixes #
Fixed the environmentIncluded flag issue for full bundle

## Proposed Changes
*
*
*

### CODE REVIEW

File: config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/environment/FullBundleCreator.java [hunk 1]
```diff
@@ -103,6 +112,15 @@ public void createFullBundle(final Pair<String, Map<String, String>> bundleEnvir
         if (!deleted) {
             LOGGER.log(Level.WARNING, "Temporary bundle file was not deleted: " + fullBundleFile.toString());
         }
+
+        // read the bundle metadata file
+        final MapType type = jsonTools.getObjectMapper(YAML_EXTENSION).getTypeFactory().constructMapType(LinkedHashMap.class, String.class, Object.class);
```
Comment (uppoju) [inline config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/environment/FullBundleCreator.java:117]: can we move this to jsonFileUtils

Comment (RaghuReddy) [reply config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/environment/FullBundleCreator.java]: @uppoju done

**Comment by RaghuReddy** [comment]
> 
> 
> JsonFileUtils should have generic methods, this method just updates one of the metadata attributes.
> Instead we can have readBundleMetadataFile and use this method in FullBundleTask

done

File: config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/util/file/JsonFileUtils.java [hunk 2]
```diff
@@ -84,4 +86,13 @@ public BundleDefinedEntities readBundleMetadataFile(final File metaDataFile) {
             throw new JsonFileUtilsException("Error reading the bundle metadata file " + metaDataFile.toString(), e);
         }
     }
+
+    public Object readBundleMetadataFile(final String bundleFolderPath, final String fileName) {
+        final File metaDataFile = new File(bundleFolderPath, fileName + METADATA_FILE_NAME_SUFFIX);
+        if (metaDataFile.exists()) {
+            final MapType type = jsonTools.getObjectMapper(YAML_EXTENSION).getTypeFactory().constructMapType(LinkedHashMap.class, String.class, Object.class);
```
Comment (nikhilahuja) [inline config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/util/file/JsonFileUtils.java:93]: Please make sure the order of the fields in the metadata is maintained after replacing it.

**Pull request merged**